getting-started/dc2-guide.rst

```diff
@@ -218,16 +219,21 @@ Processing Data with BPS
 
 The example data processing steps above assume a relatively small data volume, so running from the command line and using an sqlite APDB is appropriate.
 However, if you want to process larger data volumes, you'll need to use the Batch Processing System (BPS, :py:mod:`lsst.ctrl.bps`) and a PostgreSQL APDB.
-
-Describing how to set up a PostgreSQL APDB from scratch is beyond the scope of this guide.
 One key difference between using an sqlite APDB versus a PostgreSQL APDB is that the former is a file on disk created from scratch when running ``apdb-cli create-sql``.
-The latter requires a database to already exist, and ``apdb-cli create-sql`` turns the specified schema (via the ``namespace`` config option) in an existing PostgreSQL database into an empty APDB.
-As before, you will still need to run, e.g.,
+The latter requires a database to already exist and creation of the database is beyond the scope of this guide.
+
+As before, to create an empty sqlite APDB you will still need to run, e.g.,
 
 .. prompt:: bash
 
    apdb-cli create-sql sqlite:////path/to/my/database/apdb.sqlite3 apdb_config.py
 
+When working with a central PostgreSQL database (APDB), ``apdb-cli create-sql`` turns the specified schema (via the ``namespace`` config option) in an existing PostgreSQL database (``usdf-prompt-processing-dev`` in this example) into an empty APDB. To create a PostgreSQL APDB for a BPS configuration file that runs ``ApPipe.yaml``, these arguments are instead required to be passed into ``apdb-cli``:
+
+.. prompt:: bash
+
+   apdb-cli create-sql --namespace DESIRED_POSTGRES_SCHEMA_NAME postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl apdb_config.py
```

Comment:
Should this server be used for DC2 analysis (I assume by non-AP people)?

Reply:
It was already in the guide when I was editing. Do you know of a different server we should use and/or a generic `user@your-server-here` type of phrasing?

Reply:
No, I actually don't know the answer. Maybe @isullivan knows?

Reply:
For now I would just add a comment before or after this example pointing out that this is only for working in the development environment at the USDF, and to replace it with your postgres server address if you are running this elsewhere.

Reply:
We should think about how we want to modify our documentation overall, now that we will soon have many external users.

Reply:
I updated based on this conversation. Let me know if I need to make any more changes! Thanks everyone!